app/helpers/classifyIntlMessage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function (className) {
```

Comment:
also, given how much this was used in this pull request, how do we feel about providing a wrapped `FormattedMessage` as well, using this?

```
import { FormattedMessage: OgFormattedMessage } from 'react-intl'
import { isNil } from 'ramda'

function FormattedMessage (props) {
  return isNil(props.className)
    ? <OgFormattedMessage {...props} />
    : <OgFormattedMessage {...props} {...classifyIntlMessage(props.className) } />
}
```